pkg/kube/controllers/quarkssecret/quarkssecret_reconciler.go

```diff
@@ -19,6 +19,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	"code.cloudfoundry.org/cf-operator/pkg/credsgen"
+	bdv1 "code.cloudfoundry.org/cf-operator/pkg/kube/apis/boshdeployment/v1alpha1"
```

Comment:
We have a story in the ice box to make quarks secret a stand alone helm chart. In that case, we wouldn't want it to have any knowledge of the bdpl (boshdeployment) CRD.

I'd suggest to not use `bdv1.LabelDeploymentName` here, but instead copy a set of generic labels to the secret.
QuarksJob has the same problem: https://github.com/cloudfoundry-incubator/quarks-job/blob/594102fffff0033a3f5040a66d1b9e0c10af9906/pkg/kube/apis/quarksjob/v1alpha1/types.go#L88